nengo_viz/static/slidercontrol.js

```diff
@@ -0,0 +1,209 @@
+VIZ.SliderControl = function(min, max) {
```

Comment:
I totally agree with dropping the ```viz_*``` prefix on the files.  We'll put it on the list of things to do once it's all merged... 

And, in fact, if you want to leave this one the way it is (as kind of a marker that we should fix this in the future), I'm good with that too.